### ISSUE TITLE
Polish translation for Android app

### ISSUE THREAD

**Issue by dgolda** [owntracks/talk#42]
I'm a new user of OwnTracks Android app. I use it few days only but think it's great!
I'd like to create Polish translation for Android app.
I have problem how to translate regions and waypoints. For me it's kind of dualism or incosistency - for example I click *Regions* tab and see message "No *waypoints* "..
Or maybe I didn't catch something...

**Comment by jpmens** [comment]
I think we are guilty (at least I am guilty) of using the terms interchangeably.  I believe the correct term should be region.

**Comment by binarybucks** [comment]
Indeed. What’s now called region in most places was originally called waypoint because it denoted a location that you wanted to save. The JSON element is still called waypoint for compatibility reasons but in the GUI you can probably use the polish translation for region.

**Comment by dgolda** [comment]
OK. So I'll use "region" for both region and waypoint.
